client/core/bookie.go

```diff
@@ -511,6 +511,8 @@ func handleTradeSuspensionMsg(c *Core, dc *dexConnection, msg *msgjson.Message)
 			tracker.metaData.Host == dc.acct.host && tracker.metaData.Status == order.OrderStatusBooked {
 			// Locally revoke the purged book order.
 			tracker.revoke()
+			details := fmt.Sprintf("Order %s on market %s at %s revoked due to market suspension", tracker.token(), sp.MarketID, dc.acct.host)
+			c.notify(newOrderNote("Order Auto-revoked", details, db.WarningLevel, tracker.coreOrderInternal()))
```

Comment:
@buck54321 since these are no longer `"revoked"` (now `SubjectRevoke`), is the frontend going to have trouble updating the order status or is the subject ignored and just updates status?  I think that's what you said, but I want to be sure.

@kevinstl Depending on what @buck54321 says, I believe you can change `SubjectRevoke` to `"Order Revoked"` and use that in handleRevokeOrderMsg, and add a new subject var for `"Order Auto-revoked"` in `handleTradeSuspensionMsg`.

Otherwise, good to go here.

Reply:
I do switch on a couple of subjects on the front end, but not these. Good to go.

Reply:
Just to clarify, good to make these subject constants or just literals?

Reply:
@kevinstl This is pretty much ready to go.  Conflict resolution here would be easiest to squash first, then rebase.  One way:

```
git checkout master
git pull
git checkout revoke-order-notification
git rebase -i $(git merge-base master HEAD)
```

Then "fix" all commits but the first unless you want to include any of the messages with "squash":

```
pick e195c29e Allow for revoke notifications at variable levels
f bb613c6e Send revoke notification from callers
f 323b1f50 Revert changes to go.sum
f 1ee91dd6 Set suspension revoke to warning and make test effective
f 1939b56e Add token, market id and host to order suspension revoke notification details
f 4012cd3d Add token, market id and host to revoke order notification details
f beb19f77 Set trade suspension order revoke note subject to unique value
f f33cf135 Follow NotificationFeed pattern to listen specifically for OrderNote
f ee6aee65 Pass subject string to revoke notification verification for comparison
f 8a44a0fe Use select statement instead of for loop within goroutine with wait
f deec90c1 Ensure orderNoteFeed goroutine returns prior to test returning
f 1613f887 Use coreOrder function that does not require locked mutex
f 475dc3bd Revert changes to go.sum
```

That will have squashed all the commits down to one without rebasing on `master` yet.

Then rebase on `master` and resolve conflicts:

```
git rebase master
git mergetool # or whatever you want to do to resolve
# when all conflicts resolved and changes are staged:
git rebase --continue
```

(just remove the `t.notify` from `(*trackedTrade).revoke` afaict)

Reply:
> Just to clarify, good to make these subject constants or just literals?

Please make constants, `SubjectOrderRevoked` and `SubjectOrderAutoRevoked`.